Amplify/Core/Configuration/AmplifyConfiguration.swift

```diff
@@ -66,7 +66,8 @@ public struct AmplifyConfiguration: Codable {
                 logging: LoggingCategoryConfiguration? = nil,
                 notifications: NotificationsCategoryConfiguration? = nil,
                 predictions: PredictionsCategoryConfiguration? = nil,
-                storage: StorageCategoryConfiguration? = nil) {
+                storage: StorageCategoryConfiguration? = nil)
```

Comment:
I think we agreed on putting the closing parenthesis in the next line, not the brace itself. 
I.e.
```swift
                storage: StorageCategoryConfiguration? = nil
             ) {
```

Reply:
Updated the argument wrapping rules, braces formatting should be updated now.